integration-tests/test.sh

```diff
@@ -15,54 +16,79 @@ seccomp_opts() {
   fi
 }
 
-cp "../release/goss-linux-$arch" "goss/$os/"
-# Run build if Dockerfile has changed but hasn't been pushed to dockerhub
```

Comment:
I had to add this pushd/popd because the md5 file contains the relative path to the dockerfile, and I'm not sure how to regenerate the md5s (?). I'd be happy to do that, or alternatively leave this the way it is currently.

Reply:
It either needs to exist in both or be removed from both.

I build the images to speed up CI, but CI will fallback on building the images itself if the checksum doesn't match.

As long as the publish and this script are in sync or expect the same relative path, this mechanism will work.